### DISCUSSION TITLE
react-router-dom-compat not rendering components.

### DISCUSSION THREAD

**Discussion by Antoine-Bottin** [remix-run/react-router#9039] [v5 to v6 Migration]
Hi everyone, 

I'm acutally using  react router dom compat to slowly migrate to v6 ( we've got a huge app).
I'm following step by step the install, the `  
<BrowserRouter>
    <CompatRouter>
        <App />
    </CompatRouter>
  </BrowserRouter>`
  
 pattern .
 `</App>` component `Switch` looks like this:
 
 `        <Switch>
          <CompatRoute exact path="/" render={() => <Home />} />
          <CompatRoute path="/application/:appId" render={() => <ApplicationRouter />} />
          <Redirect to="/" />
        </Switch>
`

`Home ` component is rendering BUT `</ApplicationRouter>` doesn't. 
`<ApplicationRouter />` is as you can guess a Router too with is own `Switch`.

Is there anything that could explain that I can't render component with their own `Router`?

Thanks a lot for you help

**Comment by rodoabad** [comment]
@Antoine-Bottin  did you ever get to solve this issue?
